src/server/v2.0/handler/scan.go

```diff
@@ -48,6 +48,24 @@ func (s *scanAPI) Prepare(ctx context.Context, operation string, params interfac
 	return nil
 }
 
+func (s *scanAPI) StopScanArtifact(ctx context.Context, params operation.StopScanArtifactParams) middleware.Responder {
+	if err := s.RequireProjectAccess(ctx, params.ProjectName, rbac.ActionDelete, rbac.ResourceScan); err != nil {
```

Comment:
we define POST method in the API doc, but use `ActionDelete` to judeg permission?

Reply:
> we define a POST method in the API doc, but use `ActionDelete` to judge permission?

I was initially using `ActionUpdate`, but later on reaching an agreement with @heww to use `ActionDelete` for permission checking. But we are open to further discussion on this.